dags/webcompat-kb.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from utils.gcp import gke_command
+from utils.tags import Tag
+
+DOCS = """
+### Bugzilla to BigQuery import
+
+#### Description
+
+Runs a Docker image that fetches bugzilla bugs from
+Web Compatibility > Knowledge Base component, as well as their core
+bugs dependencies and breakage reports and stores them in BQ.
+
+The container is defined in
+[docker-etl](https://github.com/mozilla/docker-etl/tree/main/jobs/webcompat-kb)
+
+#### Owner
+
+kberezina@mozilla.com
+"""
+
+default_args = {
+    "owner": "kberezina@mozilla.com",
+    "email": ["kberezina@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2023, 9, 26),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+}
+
+
+tags = [
+    Tag.ImpactTier.tier_2,
+]
+
+every_fifteen_minutes = "*/15 * * * *"
+
+with DAG(
+    "webcompat-kb",
```

Comment:
```suggestion
    "webcompat_kb",
```

Also the file name should be renamed to `webcompat_kb.py` for consistency

Reply:
good catch, thanks!